src/main/java/frc/robot/subsystems/drive/Drive.java

```diff
@@ -139,9 +139,9 @@ public Drive(
         this::getPose,
         this::setPose,
         this::getChassisSpeeds,
-        this::runVelocity,
+        this::runVelocityOpposite,
```

Comment:
I have great concern about this change. This looks like poking at the black box without understanding things. We should not just reverse values and "see if it works". We need to understand why the underlying system is not behaving as we expect.